### DISCUSSION TITLE
Go runtime for serving templates

### DISCUSSION THREAD

**Discussion by ezlosswm** [vercel/community#2995] [Help]
### Summary

I'm trying to deploy a simple linktree clone to Vercel using the Go runtime. I know I have my project structure setup correctly and nothing seems to be wrong with the code but for some reason the only reads the index.html rather than reading from the base.html in a layouts directory. 

This would be the output: 
![image](https://github.com/vercel/community/assets/80317213/c135f84f-20a2-4268-9e99-908c24872adc)

This would be the public folder. 
```Shell
├── index.html
└── layouts
    └── base.html
```

I've tried everything that comes to mind but can't seem to figure out if its an error on my part and how can I fix it. 

### Example

https://github.com/EzlosSWM/link-in-bio

### Steps to Reproduce

1. Running the code will work correctly in dev but does not work when deployed.

**Comment by filipeapdo** [comment]
Hello @ezlosswm, any feedback or workarround? I'm trying the "same" here without much sucess...
